modules/ROOT/pages/about-sharing-assets.adoc

```diff
@@ -1,14 +1,14 @@
-= Share Assets
+= Publish Assets
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-You can use a private Exchange or an Exchange public portal to share Mule XML example code, templates, connectors, policies, APIs, API Groups, and custom information assets. Sharing assets provides a repository for code and APIs, and helps your organization access them easily, model them, and reuse them.
+Publish APIs, API Groups, templates, connectors, policies, Mule XML example code, and custom information assets to a private Exchange or an Exchange public portal. Exchange provides a repository for code and APIs, and helps your organization access them easily, model them, and reuse them.
 
-A private Exchange makes it easy to share assets with internal developers by giving them viewer, contributor, or administrator access. Additionally, assets created within a business group are automatically shared with developers within that business group.
+A private Exchange makes it easy to provide assets to internal developers by giving them viewer, contributor, or administrator access. Additionally, assets created within a business group are automatically provided to the developers of that business group.
```

Comment:
```suggestion
A private Exchange server makes it easy to provide assets to internal developers by giving them viewer, contributor, or administrator access. Additionally, assets created within a business group are automatically provided to the developers of that business group.
```
Are we distinguishing between role names and access types? They are init. capped earlier, lowercased here.

Reply:
Yes, the role names are Exchange Viewer, Exchange Contributor, Exchange Administrator, Asset Viewer, Asset Contributor, and Asset Administrator, but here we're just describing the asset types, so I'm using lower case.